src/engine/strat_engine/backstore/blockdev.rs

```diff
@@ -81,12 +88,29 @@ impl StratBlockDev {
     }
 
     /// Remove information that identifies this device as belonging to Stratis
+    ///
+    /// If self.is_encrypted() is true, destroy all keyslots and wipe the LUKS2 header.
+    /// If self.is_encrypted() is false, wipe the Stratis metadata on the device.
+    /// Both of these actions will destroy the Stratis metadata so that it is no longer
```

Comment:
I'm a little concerned about the wording here simply because if I read your suggestion, I'd assume there would be an "abnormal" way to access the Stratis metadata in the encrypted case and this is not the case. Once the LUKS2 metadata is destroyed, you will not be able to recover the Stratis metadata in a human lifetime. Would you be open to me removing that last sentence and making it more specific to each case? They will have slightly different recovery properties.

Reply:
> I'm a little concerned about the wording here simply because if I read your suggestion, I'd assume there would be an "abnormal" way to access the Stratis metadata in the encrypted case and this is not the case. Once the LUKS2 metadata is destroyed, you will not be able to recover the Stratis metadata in a human lifetime. Would you be open to me removing that last sentence and making it more specific to each case? They will have slightly different recovery properties.

Unless the no-encryption encryption option is used; in which case the Stratis metadata will be sitting somewhere on the physical device, though not at the start of the device, where blkid would look for it.

Reply:
We decided on a precondition.